public/app/types/unified-alerting-dto.ts

```diff
@@ -301,8 +301,8 @@ export interface RulerGrafanaRuleDTO<T = GrafanaRuleDefinition> {
   grafana_alert: T;
   for?: string;
   keep_firing_for?: string;
-  annotations: Annotations;
-  labels: Labels;
+  annotations?: Annotations;
```

Comment:
why do we need this change?

Reply:
Because the `annotations` field is optional in the API. The type was incorrect for a very long time

Reply:
It seems to me that `CombinedRule` was hiding that fact for a very long time. The API response doesn't contain these fields if they're empty